src/common/hooks/use-translation-wrapper.ts

```diff
@@ -0,0 +1,5 @@
+import { useTranslation as useReactI18NextTranslation } from 'react-i18next';
```

Comment:
Probably we should add an ESlint rule to ensure everybody needing access to the `useTranslation` hook, will consume it from this modules instead of the one in `react-i18next`

@celdrake already added a similar rule for `lodash` functions. PTAL at it [here](https://github.com/openshift-assisted/assisted-ui-lib/blob/23db199c5f9e8f1d7173602446cfb387236a6885/.eslintrc.js#L63)